corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -169,49 +170,8 @@ def get_form_data_schema(request, domain, form_unique_id):
     if `form_unique_id` is provided.
 
     :returns: A list of data source schema definitions. A data source schema
-    definition is a dictionary with the following format:
-    ```
-    {
-        "id": string (default instance id)
-        "uri": string (instance src)
-        "path": string (path of root nodeset, not including `instance(...)`)
-        "name": string (human readable name)
-        "structure": {
-            element: {
-                "name": string (optional human readable name)
-                "structure": {
-                    nested-element: { ... }
-                },
-            },
-            ref-element: {
-                "reference": {
-                    "source": string (optional data source id, defaults to this data source)
-                    "subset": string (optional subset id)
-                    "key": string (referenced property)
-                }
-            },
-            @attribute: { },
-            ...
-        },
-        "subsets": [
-            {
-                "id": string (unique identifier for this subset)
-                "key": string (unique identifier property name)
-                "name": string (optional human readable name)
-                "structure": { ... }
-                "related": {
-                    string (relationship): string (related subset name),
-                    ...
-                }
-            },
-            ...
-        ]
-    }
-    ```
-    A structure may contain nested structure elements. A nested element
-    may contain one of "structure" (a concrete structure definition) or
-    "reference" (a link to some other structure definition). Any
-    structure item may have a human readable "name".
+    definition is a dictionary. For details on the content of the dictionary,
+    see https://github.com/dimagi/Vellum/blob/master/src/datasources.js
```

Comment:
It's on my todo list to improve this documentation. Since vellum is currently the only consumer I'm planning to do it there.